### DISCUSSION TITLE
Update "Gallow" medium anagram clue

### DISCUSSION THREAD

**Discussion by VioRS** [runelite/runelite#16324] [Ideas]
![LTU6SiS](https://user-images.githubusercontent.com/40777930/221327422-03576f91-d22e-42fa-967a-4ddd155f65f0.png)

"Vinery in the Great Kourend" is a bit awkward and doesn't really give any useful information. Recommend changing to "Vinery south-east of Hosidius".

**Comment by Nightfirecat** [comment]
I've created #16331 to track updating this clue location text.
